### PR TITLE
Windows Color

### LINKED ISSUE #64: Colours don't work in CMD and PowerShell
PowerShell

### PR DESCRIPTION
**Pull request by mattn**
Fixes #64

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func visit(url *url.URL) {
 	if err != nil {
 		log.Fatalf("unable to connect to host %vv %v", raddr, err)
 	}
-	fmt.Printf("\n%s%s\n", color.GreenString("Connected to "), color.CyanString(raddr.String()))
+	fmt.Fprintf(color.Output, "\n%s%s\n", color.GreenString("Connected to "), color.CyanString(raddr.String()))
```
Comment (davecheney) [inline main.go]: Can you please create a top level helper function called `printf` that handles this, then rewrite all the calls to fmt.Fprintf to use this.

**Comment by mattn** [comment]
Sorry, w.Close() is invalid. Will fix also.

**Comment by mattn** [comment]
fixed.

**Comment by mattn** [comment]
Seems still bad

**Comment by mattn** [comment]
fatih/color#Output is writer given from mattn/go-colorable#NewColorableStdout().

it's return it self if the os.Stdout is not terminal.

https://github.com/mattn/go-colorable/blob/master/colorable_windows.go#L78-L91

Still you want? I'm okay to change.

**Comment by davecheney** [comment]
Let's get it working with a wrapper first then we can optimise it

**Comment by davecheney** [comment]
Thank you!

**Pull request merged**